src/components/ScholarshipDetailCard.jsx

```diff
@@ -118,52 +118,63 @@ export default function ScholarshipDetailCard({ scholarship }) {
     text: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
     bottom: PropTypes.bool,
     top: PropTypes.bool,
+    link: PropTypes.bool,
   };
   DetailCardCell.defaultProps = {
     bottom: false,
     top: false,
+    link: false,
   };
 
   return (
     <Card className={classes.detailSection}>
       <Typography variant="h4">{name}</Typography>
-      <Typography variant="h5">{organization}</Typography>
+      <Typography variant="h5" gutterBottom>
+        {organization}
+      </Typography>
+
       <Box className={classes.actionSection}>
         <Button
           component={Link}
           href={website}
           variant="contained"
           color="primary"
           className={classes.applyBtn}
-          startIcon={<Send />}>
```

Comment:
I'm realizing that this might be why `prettier` by default likes to have `>` on separate lines by themselves. The diff is cleaner when you add or remove attributes to an element with lots of attributes like this one. 🤔